decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Decorator < Person
```

Comment:
- [ ] Please make sure that the `Decorator` inherits from the `Nameable` class, as per [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#:~:text=Make%20sure%20that%20it%20inherits%20from%20Nameable.) requirement.